tests/test_net_connect.js

```diff
@@ -61,8 +60,7 @@ describe('`enableNetConnect()`', () => {
 
     await assertRejects(
       got('https://example.test/'),
-      Error,
-      'Nock: Disallowed net connect for "example.test:80/"'
```

Comment:
This is a good example of where the test was passing when it had a bug in it - the test expects port 80 when it was actually throwing an error for port 443 (https).